cg/services/pacbio/metrics/metrics_parser.py

```diff
@@ -23,43 +25,48 @@ def __init__(self, smrt_cell_path: Path) -> None:
         self.base_calling_report_file: Path = get_file_in_directory(
             directory=self.report_dir, file_name=PacBioDirsAndFiles.BASECALLING_REPORT
         )
-        self.hifi_metrics: HiFiMetrics = self.parse_attributes_to_model(
+        self.hifi_metrics: HiFiMetrics = self.parse_report_to_model(
```

Comment:
This seems to repeat the mistakes we discussed earlier today. The parser should not accept a path in the constructor.

Reply:
This is probably out of scope for this PR, but it seems like you are repeating the exact same design mistakes in the code for the new instruments

Reply:
I was planning to fix that in another PR. Let's first complete all the metrics that we need